book/website/ethical-research/activism/activism-env-impact.md

```diff
@@ -0,0 +1,166 @@
+
+
+For this reason, it is important to consider the environmental impact of the computational work you and your colleagues and collaborators do, including having awareness of the possible scale of it in relation to other activities you undertake as part of day-to-day life. With this awareness, it follows that we should all aim to reduce our carbon footprint for our work activities.
+
+This guide helps data and computational scientists to better understand the environmental impacts of their work and projects, as well as providing suggestions for reducing the impact of it.
+
+The topic is not new, it has for example been discussed during CW23 [link to raspberry blog post, when it is available].
+
+(er-activism-env-impact-understanding)=
+## Understanding the problem
+
+Most of us are aware of the general context of our planet being subject to climate change that results largely from human influence, and how this poses a great threat to our society. However, in case you are not, and indeed to emphasise this crucial background, we’ll summarise this background.
+
+(er-activism-env-impact-climate-change-sumarised)=
+### Climate change summarised
+
+Human activities, notably the burning of fossil fuels like oil, gas or coal, to generate electricity and to power cars (amongst many other tasks) release greenhouse gases into the atmosphere. These gases, such as carbon dioxide, trap heat in the atmosphere, which in turn raises the temperature of the surface of the Earth. While not linear, over time, [the global average temperature rises](http://climate.nasa.gov/vital-signs/global-temperature/).
+
+
+```{figure} ../../figures/global_warming.*
+---
+height: 500px
+name: Simplified view of global warming
+alt: Simplified view of global warming.
+---
+Simplified view of global warming.
+```
+
+(er-activism-env-impact-rise-in-temperature)=
+### Rise in global temperatures illustrated
+
+The rise in this global average temperature is strikingly illustrated by the ‘Warming Stripes’ (‘#ShowYourStripes’) project, a minimalist-style data visualisation created by Ed Hawkins. Temperatures are indicated as stripes in blue hues, representing cooler temperatures, or red hues, representing warmer ones. The general trend shows a progression from there being more blue on the left (towards the past) to more red on the right (towards the present). In short, the planet is, overall, warming!

```

Comment:
Hi, I wrote this paragraph, in a bit of a rush during the SSI CW23 Hack Day due to trying to get as much done towards the deadline for presentations. I am thinking now that it could do with some clarifications. What do people think about this update:

```suggestion
The rise in this global average temperature is strikingly illustrated by the ‘Warming Stripes’ (‘#ShowYourStripes’) project, a minimalist-style data visualisation created by Ed Hawkins. Average Earth surface temperatures are indicated as stripes in blue hues, representing cooler temperatures, through to red hues, representing warmer ones, where each stripe covers a single year.

The general trend shows a progression, over the past few centuries or so, from there being more blue on the left (towards the past) to more red on the right (towards the present). In short, the surface of our planet is, overall and year-on-year, warming!

```